Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM hashicorp/packer:latest
```

Comment:
should we not use this (https://github.com/hashicorp/setup-packer) instead of building in a container?

Reply:
You are correct. My thought was that a container would let people run this locally as well if needed (SE/CS teams).

Reply:
Should be the same without the container as its just invoking `packer build`. The actual building happens in GCP anyway.